assets/css/uikit-rtl.css

```diff
@@ -3033,6 +3033,7 @@ select.uk-form-width-xsmall {
   justify-content: center;
   align-items: center;
   line-height: 0;
+  white-space: nowrap;
```

Comment:
This van be achieved by adding uk-text-nowrap to the badge component. I'll change it. Good catch!